backend/tests/features/carrinho_compras.feature

```diff
@@ -0,0 +1,31 @@
+Feature: carrinho de compras
+    As eu sou um cliente
+    I want escolher os itens que desejo comprar
+    So that eu possa efetuar e receber meu pedido
+
+Scenario: inserir um item no carrinho de compras
+    Given O array "carrinho_compras" do "users/1"
```

Comment:
This setence means nothing.
It could be 
`Given a lista de "itens" no "carrinho de compras" do "usuario1" está vazio`